appveyor.yml

```diff
@@ -0,0 +1,45 @@
+environment:
+  password:
+    secure: uOcv5TOm/CgwOmiLQhNX+g==
```

Comment:
How does it work? I.e. to deploy to pypi one needs an username and a password, right?